rules/prefer-includes.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const isIndexOfCallExpression = node => {
+	if (node.type !== 'CallExpression') {
+		return false;
+	}
+
+	const property = node.callee.property;
+
+	return property.name === 'indexOf';
```

Comment:
```js
function indexOf(input) {
  // ...
}

indexOf(foo);
```

But in this implementation, you assume that any CallExpression's callee is a `MemberExpression`, which will cause a crash before you even check whether the property is `indexOf`. Add `indexOf(foo)` and `a()` in your valid tests and the problems should be made clear right awaya IIRC.